app/common/locales/en.json

```diff
@@ -1119,7 +1119,7 @@
                 }
             },
             "twitter": {
-                "intro_step_1": "Step 1: Create a new Twitter application",
+                "intro_step_1": "Step 1: Create a new Twitter application: <br />Twitter applications may take a few hours or days to be approved by Twitter.<br />Please be aware of this if you need this data quickly.",
```

Comment:
@Jenniline One more thing, could you remove the " in the end of the sentence?

Reply:
Nevermind, I missed there are a " in the beginning, forget my comment! Merging! Thank you for your contribution! 🎉